clients/js/lib/serializer/proto_serializer.js

```diff
@@ -0,0 +1,10 @@
+function createProtobufSerializer() {
+
+    function serialize(data) {
+        // TODO://Return the serialized protobuf data
+    }
+
+    return { serialize };
```

Comment:
can't we write this as return 
```
return function serialise(data){
...
}
```
instead of returning function inside an object ?

Reply:
wan't to use serialize as a common function for all serialisers